installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -168,6 +168,8 @@ spec:
                 secretKeyRef:
                   name: keptn-api-token
                   key: keptn-api-token
+            - name: CLI_DOWNLOAD_LINK
+              value: {{ .Values.bridge.cliDownloadLink | default (print "https://github.com/keptn/keptn/releases/tag/" .Chart.AppVersion) }}
```

Comment:
I have tested this with @agrimmer

It basically uses the download link if it is available, else it builds a download-link that points to the github releases page, e.g., https://github.com/keptn/keptn/releases/tag/0.7.1